pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/design/xml/TooManyMethods.xml

```diff
@@ -110,6 +110,40 @@ public class OuterClass {
         @Override
         public void method7() {}
     }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>#3729 TooManyMethods ignores "real" methods which are named like getters or setters</description>
+        <expected-problems>0</expected-problems>
```

Comment:
Shouldn't we expect here a violation?

The class has 22 methods, which are `> 10`... It looks like the trick with `count(../Block/BlockStatement)<=1)` didn't work out?